layout/js/map.js

```diff
@@ -29,6 +29,19 @@ var animate_resize_map = function(data) {
   });
 };
 
+var animate_update_tooltips = function(data) {
+  $.each(data, function() {
+    // create the tooltip text
+    var newtip = this.state_name + ': ' + this.value + ' million gallons per day'
+    // this section doesn't work. trying to assign that text to the hovertext 
+    // call for onmousemove for each object, but haven't found the way. help!
+    var newtipobj = {
+      "onmousemove": "hovertext('" + newtip + "', evt);"
+    };
+    $("#" + this.state_name + "-mouseover").mousemove = hovertext(newtip);
```

Comment:
I think changing the function on the element (what I think is being done here) is maybe not the best way to tackle it. Instead, the hovertext always just takes the state name and the event (`evt`), but gets the other info based on three variables that are set globally - the category, the time, and the value. Whenever a function changes those variables, it also needs to wipe and recalculate the text string that is displayed. Seems like this would work?